torchao/prototype/smoothquant/core.py

```diff
@@ -0,0 +1,167 @@
+from dataclasses import dataclass
+from typing import Optional
+import torch
+import torch.nn.functional as F
+from torch.ao.quantization import PerChannelMinMaxObserver, MovingAverageMinMaxObserver
+from torchao.quantization.quant_primitives import (
+    MappingType,
+    ZeroPointDomain,
+)
+from torchao.quantization.observer import (
+    AffineQuantizedObserverBase, PerRow
+)
+
+
+class SmoothQuantObserver(AffineQuantizedObserverBase):
```

Comment:
it seems that smooth quant observer is not related to AQT, probably don't need to inherit from `AffineQuantizedObserverBase`?

Reply:
Thanks. It's changed.